templates/abilities.html

```diff
@@ -450,6 +469,7 @@ <h2>Abilities</h2>
                 this.fieldErrors = [];
                 this.selectedAbility = ability;
                 this.showAbilityModal = true;
+                console.log(this.selectedAbility)
```

Comment:
Feel free to remove the console.log, helps make it easier to debug when a stray error comes up.